try:
    fin = open('bad_file')
except:
    print('Something went wrong.')